README.md

```diff
@@ -143,6 +144,20 @@ component.
 #### Inheritance
 
 The properties of the [`Box`](#box) component are also available.
+### Avatar
+
+[View story source](https://github.com/balena-io-modules/rendition/blob/master/src/components/Avatar/story.js)
+
+#### Props
+
+| Name   | Type   | Default   | Required   | Description   |
+| ------ | ------ | --------- | ---------- | ------------- |
+| `email`| `string`| - | - | The email to fetch the gravatar |
```

Comment:
Let's just have `src` and do the transformation of email -> gravatar URL as part of the rendition client for a start

Reply:
IMHO it would be worth to put the `email` prop back in the future, otherwise we will end up implementing the getGravatar function in all projects

Reply:
@JSReds let's start without it, and then decide if it is something that will be used in multiple places. Maybe we'll allow uploading photos to the DB, in which case we won't use gravatar.